wiki/People/Tournament_Committee/en.md

```diff
@@ -26,13 +34,119 @@ Their responsibilities include:
+
+### Public Communication
+
+Public communication by the Tournament Committee is handled through several channels. The purposes for these sometimes overlap, but all are official sources of information kept up-to-date by the Tournament Committee.
+
+The Tournament Committee maintains an [account on X](https://x.com/osu_tcomm) (formerly Twitter) where the committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
+
+The [Community Tournament Status Tracker](https://docs.google.com/spreadsheets/d/1bV5MyrJZI0F52Bx9EcYxdfRu8qGnhWRBKdXHK9uPbO0/edit?usp=sharing) is a publicly viewable spreadsheet detailing the status of all tournaments that have requested official support. This sheet shows all tournaments dating back to June 20th, 2023 and displays their status through the official support process, including whether a badge was applied or denied.
+
+The [Official Tournament Support Updates thread](/community/forums/topics/1715676) is a pinned thread in the [Tournaments & Contests forum](/community/forums/126) which contains all updates to the Official Tournament Support wiki dating back to February 1st, 2023. This thread is updated and bumped every time a change is made to the Official Tournament Support wiki. This thread also contains links to the Official Support wiki and the Community Tournament Status Tracker.
+
+### Adding New Members
+
+New members to the Tournament Committee must meet the following basic criteria before being considered for the position:
+- Be at least 18 years old
+- Pass a security check
+
+Whether a prospective member applies via email or is brought forward by an existing member, the process is the same. The Tournament Committee holds an in-depth discussion on the candidate. Special attention is paid towards their activity in the tournament scene, stature, community contributions, and how they would benefit the committee. Following said discussion, an anonymous vote is held to determine whether to add the candidate to the committee.
+
+### Votes
+
+The Tournament Committee frequently makes use of anonymous voting when making important decisions. Typical uses are listed above, but those are not fully inclusive. The voting requirements also differ based on the use case. All typical votes also require “strict participation,” meaning that if 75% of the Tournament Committee does not participate by the vote deadline, voting is extended by 24 hours. Further, all votes require at least an absolute majority, at least 50%, for action to be taken. Votes on many topics have the same requirements, but there are slight differences on some common topics listed below.
```

Comment:
- we don't use "smart quotes", which appear when copy-pasting from google docs or other word processors
- comma on the outside (wikipedia style https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#logical-quotation)

```suggestion
The Tournament Committee frequently makes use of anonymous voting when making important decisions. Typical uses are listed above, but those are not fully inclusive. The voting requirements also differ based on the use case. All typical votes also require "strict participation", meaning that if 75% of the Tournament Committee does not participate by the vote deadline, voting is extended by 24 hours. Further, all votes require at least an absolute majority, at least 50%, for action to be taken. Votes on many topics have the same requirements, but there are slight differences on some common topics listed below.
```